pkg/lib/DynamicListForm.jsx

```diff
@@ -84,11 +126,14 @@ export class DynamicListForm extends React.Component {
         );
     }
 }
+
 DynamicListForm.propTypes = {
     emptyStateString: PropTypes.string.isRequired,
     onChange: PropTypes.func.isRequired,
     id: PropTypes.string.isRequired,
     itemcomponent: PropTypes.object.isRequired,
     formclass: PropTypes.string,
     options: PropTypes.object,
+    validationFailed: PropTypes.array.isRequired,
+    onValidationChange: PropTypes.func.isRequired,
```

Comment:
I see we don't use this component anywhere in cockpit, but in podman. As "isRequired" is an API change, that will have to be adjusted when bumping the lib import. I suppose that's what you wanted to do, just making sure?

Reply:
That's what I wanted initially, but after few iterations of this PR, validation was basically made optional, so 'isRequired' is not wanted anymore. Thanks for pointing that out